flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/slotmanager/SlotManagerConfigurationTest.java

```diff
@@ -65,8 +64,30 @@ public void testPreferLegacySlotRequestTimeout() throws Exception {
         final SlotManagerConfiguration slotManagerConfiguration =
                 SlotManagerConfiguration.fromConfiguration(configuration, WorkerResourceSpec.ZERO);
 
-        assertThat(
-                slotManagerConfiguration.getSlotRequestTimeout().toMilliseconds(),
-                is(equalTo(legacySlotIdleTimeout)));
+        assertThat(legacySlotIdleTimeout)
+                .isEqualTo(slotManagerConfiguration.getSlotRequestTimeout().toMilliseconds());
+    }
+
+    @Test
+    void testComputeMaxTotalMemory() throws Exception {
```

Comment:
We also need a test for rounding down division.

Reply:
What does `test for rounding down division` mean? If it means that the memory should be tested for not being divisible by `numSlots`, then `MemorySize.ofMebiBytes(totalTaskManagerMB).getBytes()` here exactly cannot be divided by `maxSlotNum `.

Reply:
Thanks for explaining that, it is indeed the case that I want.